library/network/src/lib/network/susefirewall2.rb

```diff
@@ -261,14 +261,6 @@ def main
 
     # <!-- SuSEFirewall GLOBAL FUNCTIONS USED BY LOCAL ONES //-->
 
-    # Returns whether records in variable should be written one record on one line.
-    # @return [Boolean] if wolpr
-    def WriteOneRecordPerLine(key_name)
-      return true if key_name.nil? && key_name == ""
-
-      Builtins.contains(@one_line_per_record, key_name)
```

Comment:
It seems that the `@one_line_per_record` value is not used any more. Please remove it as well.